static/js/admin/src/action/ExportMetadataXMLBatchAction.js

```diff
@@ -58,7 +56,6 @@ define('ExportMetadataXMLBatchAction', [ 'jquery', 'AbstractBatchAction', "tpl!.
 		
 		this.$form.submit(function(e){
 			var email = $("#xml_recipient_email", self.$form).val();
-			var includeChildren = $("#export_xml_include_children", self.$form).prop("checked");
```

Comment:
We still need the boolean from this checkbox, right?  It is just going in as a parameter now rather than changing the endpoint used

Reply:
ok, sorry, i thought you had asked me to remove it